docs/api/0.14.22/cli/agent.md

```diff
@@ -155,13 +155,13 @@ Options:
   --help  Show this message and exit.
 
 Commands:
-  install  Generate a supervisord.conf file for a Local agent
+  install  Generate a deployment.yml file for a Kubernetes Agent
   start    Start a Kubernetes agent
 ```
 
 ### kubernetes install
 ```
-Generate a supervisord.conf file for a Local agent
+Generate a deployment.yml file for a Kubernetes Agent
```

Comment:
If you could change the rest of these to read "Generate a deployment.yml file for a Kubernetes agent."